Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMDOMWindow.cpp

```diff
@@ -1074,7 +1076,10 @@ gboolean webkit_dom_dom_window_webkit_message_handlers_post_message(WebKitDOMDOM
         return FALSE;
 
     auto promise = WebCore::DeferredPromise::create(*globalObject);
-    auto result = handler->postMessage(WebCore::SerializedScriptValue::create(String::fromUTF8(message)), adoptRef(*(promise.leakRef())));
+    JSRetainPtr<JSStringRef> jsString(Adopt, JSStringCreateWithUTF8CString(message));
+    JSValueRef jsStringValue = JSValueMakeString(toRef(globalObject), jsString.get());
+
+    auto result = handler->postMessage(*globalObject, toJS(globalObject, jsStringValue), adoptRef(*(promise.leakRef())));
```

Comment:
I am not sure if there is a better way to do this; I also didn't find a way to test this code either beyond making sure it builds.